### PR TITLE
added else so that the CONNECTING event is not emitted after an error…

### PR DESCRIPTION
**Pull request by jhaugh42**
… is handled

### CODE REVIEW

File: test/unit/busterbunny-tests.js [hunk 1]
```diff
@@ -387,4 +387,137 @@ describe("busterbunny.js", function() {
...
+
+        bb.on(bb.EVENTS.RECONNECTING, function() {
+            assert.ok('RECONNECTING was emitted.');
+            done();
+        });
+
+        connectWithError(new Error('whatever'));
+    });
+
+    it('should not emit a CONNECTING event when _amqp.connect calls back with an error', function(done) {
```
Comment (jhaugh42) [inline test/unit/busterbunny-tests.js]: should emit CONNECTING ... without an error

File: test/unit/busterbunny-tests.js [hunk 2]
```diff
@@ -387,4 +387,137 @@ describe("busterbunny.js", function() {
...
+
+        bb.on(bb.EVENTS.CONNECTING, function() {
+            assert.ok('CONNECTING was emitted.');
+            done();
+        });
+
+        connectWithoutError(null, {});
+    });
+
+    it('should not emit a RECONNECTING event but not a CONNECTING event when _amqp.connect calls back with an error', function(done) {
```
Comment (jhaugh42) [inline test/unit/busterbunny-tests.js]: should emit a RECONNECTING but not a CONNECTING event

File: test/unit/busterbunny-tests.js [hunk 3]
```diff
@@ -387,4 +387,137 @@ describe("busterbunny.js", function() {
...
+            assert.strictEqual(connectingCalled, false);
+            assert.strictEqual(reconnectingCalled, true);
+            done();
+        });
+
+        connectWithError(new Error('whatever'));
+        bb.emit('verify');
+    });
+
+    it('should not emit a CONNECTING event but not a RECONNECTING event when _amqp.connect calls back without an error', function(done) {
```
Comment (jhaugh42) [inline test/unit/busterbunny-tests.js]: should emit a CONNECTING

**Comment by binaryalchemist** [comment]
Please pull master and fix conflicts.

**Pull request merged**